src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/debtposition/wfingestion/DebtPositionIngestionFlowWFImpl.java

```diff
@@ -57,14 +61,25 @@ public void ingest(Long ingestionFlowFileId) {
 
     log.info("Lock successfully acquired for ingestionFlowFileId {}", ingestionFlowFileId);
     InstallmentIngestionFlowFileResult ingestionResult = processFile(ingestionFlowFileId);
-    boolean success = StringUtils.isEmpty(ingestionResult.getErrorDescription());
+
+    String error = synchronizeIngestedDebtPositionActivity.synchronizeIngestedDebtPosition(ingestionFlowFileId);
+
+    String errorDescription = StringUtils.join(
+        Arrays.asList(StringUtils.isEmpty(ingestionResult.getErrorDescription()) && !StringUtils.isEmpty(error)
```

Comment:
Let's not create a List just to concat 2 string.
let's explicitly use if condition instead

ps: seeing the test, I change my mind, let's always prepend the string
"There were errors during the synchronization of the ingested Debt Position:"
on the additionalErrors if any and concat it to the error_description, if present, with an empty line, eg:
```
ERROR DURING SYNC

There were errors during the synchronization of the ingested Debt Position:
...
```